pkg/flowaggregator/flowrecord/record.go

```diff
@@ -228,6 +230,12 @@ func GetFlowRecord(record ipfixentities.Record) *FlowRecord {
 	if egressIP, _, ok := record.GetInfoElementWithValue("egressIP"); ok {
 		r.EgressIP = egressIP.GetStringValue()
 	}
+	if isL7, _, ok := record.GetInfoElementWithValue("isL7"); ok {
+		r.IsL7 = isL7.GetStringValue()
```

Comment:
A general comment: since `IsL7` is a boolean value, I suggest we change to use `BooleanInfoElement` for this field